CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contribution Guidelines
+Hello, and thank you for your interest in contributing to openScope! Please take a minute to review our Contribution Guidelines, as this will result in getting your issue resolved or pull request merged faster.
+## Getting set up
+openScope uses multiple tools for development, the basics being Git and Node. Getting these set up are covered in the quick start guide in README.md. To contribute, you will also need an IDE and eslint (`npm install -g eslint`). After this, you are ready to create a branch and get developing!
```

Comment:
please remove the `npm install` part.  I tend to discourage global installs of `eslint` because different projects may depend on different `eslint` versions.  It's better to let the project decide versions and not fallback to a global version.

Reply:
done